### PR TITLE
feat: projects list

### PR DESCRIPTION
**Pull request by kkatusic**

### CODE REVIEW

File: apps/platform/messages/en.json [hunk 1]
```diff
@@ -89,5 +89,27 @@
...
+    "neighborhood": "By neighborhood",
+    "filterNeighborhood": "Filter by Neigbourghood",
+    "projectsMap": "Projects on the map"
+  },
+  "projectCard" : {
+    "funded": "funded"
+  },
+  "project" : {
+    "funded": "funded of €{amount} goal",
+    "bakers": "bakers",
```
Comment (b0rza) [inline apps/platform/messages/en.json:106]: Typo, unless we're baking 🥐 😆 

```suggestion
    "backers": "backers",
```

File: apps/platform/src/app/helpers/projectHelper.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * Determines the background color class for a project's progress bar
+ * based on the given progress percentage.
+ *
+ * @param {number} progressPercentage - The current progress of the project as a percentage (from 0 to 100).
+ * @returns {string} - A string representing the CSS class name for the background color.
+ *
+ * - Returns `'bg-[#C9767B]'` for progress between 1% and 33%.
+ * - Returns `'bg-[#E2CB55]'` for progress between 34% and 66%.
+ * - Returns `'bg-[#09CE78]'` for progress between 67% and 100%.
+ * - Returns `'bg-green'` if progress is 0% or any other value outside the 1-100 range.
```
Comment (b0rza) [inline apps/platform/src/app/helpers/projectHelper.ts:11]: Maybe add these to tailwind config so we can do eg.
```suggestion
 * - Returns `'bg-progressLow'` for progress between 1% and 33%.
 * - Returns `'bg-progressMedium'` for progress between 34% and 66%.
 * - Returns `'bg-progressHigh'` for progress between 67% and 100%.
 * - Returns `'bg-green'` if progress is 0% or any other value outside the 1-100 range.
```

File: apps/platform/src/app/config/config.ts [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+	'Plokite',
+	'Skalice',
+	'Manus',
+	'Firule',
+	'Šine',
+	'Smrdečac',
+	'Sirobuja',
+	'Stobreč',
+	'Žrnovnica',
+	'Vranjic',
```
Comment (b0rza) [inline apps/platform/src/app/config/config.ts:32]: Order alphabetically, unless you need this order.
```suggestion
		'Bačvice',
		'Bol',
		'Brda',
		'Firule',
		'Gripe',
		'Kman',
		'Kocunar',
		'Lovret',
		'Manus',
		'Meje',
		'Mertojak',
		'Plokite',
		'Pujanke',
		'Šine',
		'Sirobuja',
		'Skalice',
		'Smrdečac',
		'Spinut',
		'Stobreč',
		'Sućidar',
		'Trstenik',
		'Varoš',
		'Visoka',
		'Vranjic',
		'Žnjan',
		'Žrnovnica',
```

File: apps/platform/src/app/components/project/ProjectCard.tsx [hunk 4]
```diff
@@ -2,59 +2,49 @@ import Image from 'next/image';
...
 
 interface ProjectCardProps {
 	className?: string;
+	progressPercentage: number;
 }
 
-export default function ProjectCard({ className }: ProjectCardProps) {
+export default function ProjectCard({
+	className,
+	progressPercentage,
```
Comment (b0rza) [inline apps/platform/src/app/components/project/ProjectCard.tsx:14]: nitpick: to have it more concise when typing out component:
```suggestion
	progress,
```

File: apps/platform/messages/hr.json [hunk 5]
```diff
@@ -88,5 +88,27 @@
   "languages" : {
     "en": "Engleski",
     "hr": "Hrvatski"
+  },
+  "projects" : {
+    "title": "Projekti",
+    "map": "Mapa",
```
Comment (b0rza) [inline apps/platform/messages/hr.json:94]:
```suggestion
    "map": "Karta",
```

File: apps/platform/messages/hr.json [hunk 6]
```diff
@@ -88,5 +88,27 @@
...
+    "board": "Ploča",
+    "neighborhood": "Po kvartovima",
+    "filterNeighborhood": "Filter po kvartovima",
+    "projectsMap": "Projekti na mapi"
```
Comment (b0rza) [inline apps/platform/messages/hr.json:98]:
```suggestion
    "projectsMap": "Projekti na karti"
```

**Review by b0rza — APPROVED** [review]
👍 minor nitpicks
please fix them in another PR so we don't block the pipeline

File: apps/platform/messages/en.json (hunk 1 above)
Comment (kkatusic) [reply apps/platform/messages/en.json:106]: resolved

**Pull request merged**